User.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HelloWorld
+{
+    public class User
+    {
+        public string FirstName { get; set; }
+        public string MiddleName { get; set; }
+        public string LastName { get; set; }
+        public List<string> PhoneNumbers { get; set; }
+
+        public enum PhoneTypes
```

Comment:
better to place enum within the namespace but outside the class. 
this will reduce imports number and simplify accessing enum itself